src/extensionmanager.ts

```diff
@@ -336,14 +368,17 @@ export class ExtensionManager implements Disposable {
 
     //Set up the initial status bars
     private initializeStatusBars() {
-        if (this.EnsureInitialized(RepositoryType.ANY)) {
+        if (this.EnsureInitializedForTFVC()) {
             this._teamServicesStatusBarItem.command = CommandNames.OpenTeamSite;
-            this._teamServicesStatusBarItem.text = this._serverContext.RepoInfo.TeamProject;
+            this._teamServicesStatusBarItem.text = this._serverContext.RepoInfo.TeamProject ? this._serverContext.RepoInfo.TeamProject : "<none>";
             this._teamServicesStatusBarItem.tooltip = Strings.NavigateToTeamServicesWebSite;
             this._teamServicesStatusBarItem.show();
-            // Update the extensions
-            this._teamExtension.InitializeStatusBars();
-            //this._tfvcExtension.InitializeStatusBars();
+
+            if (this.EnsureInitialized(RepositoryType.ANY)) {
+                // Update the extensions
+                this._teamExtension.InitializeStatusBars();
+                //this._tfvcExtension.InitializeStatusBars();
```

Comment:
Sorry, didn't see your comment (nor did I expect you to review on Saturday night).  :smile: We leave some comments in like this so we can easily find the place where we might need to do something in the future.  In this case, if the TFVC extension would ever provide status bar items, this is where we'd do it.